### DISCUSSION TITLE
Why is it reccomended to run remove.sh on a 10 minute interval?

### DISCUSSION THREAD

**Discussion by viridivn** [Enkidu-6/tor-ddos#24] [Q&A] [answered]
Apologies for the basic question. Wouldn't it be better to let bad relays wait the full 12 hours? I always get at least several hundred IPs caught in my filter list before it resets, but according to compare.sh, never more than 2 of them are actually relays, which is weird because there's no other reason I'd be connecting to so many IPs. I'm still dropping ntors pretty frequently and showing as overloaded in the relay search

**Comment by Enkidu-6** [comment, marked as answer]
Hi. It's a valid question. The reason I said that is because in the new version of the rules, we're allowing only two connections per IP address which theoretically should be more than enough. But in case of relays that are running two instances of Tor on the same IP address, chances of them getting caught in the block list is higher. By removing them, they'll get a second chance. But the next rule will stop them from making more than 4 connections anyway.

As for the interval of running the cron, it's totally up to you. I personally don't run it as often and share your view but I thought people should have a choice. Just keep an eye on the list by running the compare.sh and decide for yourself.

Regarding the NTor drops, we had a discussion about how to stop that [here](https://github.com/Enkidu-6/tor-ddos/discussions/19). More precisely from [this point](https://github.com/Enkidu-6/tor-ddos/discussions/19#discussioncomment-4169310) down.

**Reply by viridivn** [reply]
Thanks - I'll check out that discussion, and mess around with the cron settings - I've put it on twice per hour now.
